src/__tests__/index.test.tsx

```diff
@@ -105,7 +113,9 @@ describe('EppoReactNativeClient integration test', () => {
     const mockConfigStore = td.object<EppoAsyncStorage>();
     const mockLogger = td.object<IAssignmentLogger>();
     td.when(mockConfigStore.get(flagKey)).thenReturn(null);
-    const client_instance = new EppoReactNativeClient(mockConfigStore);
+    const client_instance = new EppoReactNativeClient({
+      flagConfigurationStore: mockConfigStore,
```

Comment:
Signature of the core repository constructor has changed from an argument list to an options object

Reply:
mea culpa